src/components/Forms/Venue/index.tsx

```diff
@@ -27,7 +27,7 @@ export default function VenueForm() {
         editVenueMutationIsSuccess,
         error,
         reset
-    } = useVenueForm()
+    } = useVenueForm(data ?? undefined)
```

Comment:
This is what I mean. (this is the way I had it originally). Causes a type error when we follow this pattern on Events hook.

Reply:
@velveteen-rever we should not be passing data here. The data is passed via the reset in all forms because it will not be there are first render. You'd only pass it here if you were passing the  data via a prop to the component, which we aren't anymore.

Reply:
I added it back because otherwise isEditing does not get set to true because it is bound to the argument.

Reply:
Fixed now.  I bound isEditing to an argument that only gets passed when query param is present.